.env.example

```diff
@@ -0,0 +1,7 @@
+BUNDLE_GEMFILE=
```

Comment:
Why are you customizing BUNDLE_GEMFILE variable?

Reply:
Hi! I have add this, because sometimes bundler may raise exceptions when you work with docker on MacOS. As Viktor works with MacOS i decided to add this to prevent exceptions

Reply:
I have this project configured on my MacOS and it works well without such key. So let's remove this.